data/competencies.yml

```diff
@@ -731,6 +731,18 @@
   area: leadership
   domain: null
 
+- id: de580f8a-633d-43bb-b970-9b2987e07b86
+  summary: Actively fosters an inclusive team culture
+  examples:
+    - Celebrates good work publicly and encourages the team to do the same
+    - Spots problems between team members and helps to resolve them
```

Comment:
I am not sure about "helps to resolve them" - maybe adding "escalating as necessary" - there are some conflicts people may not feel confident resolving themselves, maybe if they are a younger individual and the conflict is between two older ones, or more senior ones (or myriad of other things).

Reply:
Does this require an acknowledgement of power and privilege?

For example if I identify issues that do not impact me but then escalate to someone I know will not action it, that could be seen as performative? 

Would be nice to try to get people to act where they have the privilege to do so without also creating a criteria that, as you say, could force people to resolve systemic issues that impact themselves.

Reply:
It almost feels like saying "use your privilege to stand up for others" is what we'd like people to do. 

It might be that the mediating is viewed partly as the leadership thing (leaders often mediate conflicts) rather than it being inclusivity specific. Maybe the "conflict resolution" should go into leadership / communications if that's the case?

Reply:
> I am not sure about "helps to resolve them" - maybe adding "escalating as necessary" - there are some conflicts people may not feel confident resolving themselves, maybe if they are a younger individual and the conflict is between two older ones, or more senior ones (or myriad of other things).

It's not required that you meet _all_ examples for a competency ([or any of them if there's a different way to do so](https://engineering-progression.ft.com/competencies/how-to-use/#examples)), so if this isn't a thing that somebody is comfortable with then they can find other ways to demonstrate the competency 🙂 I think the addition of "escalating as necessary" could go in, something like:

```suggestion
    - Spots problems between team members and helps to resolve them or escalate them as appropriate
```

---

> Does this require an acknowledgement of power and privilege? For example if I identify issues that do not impact me but then escalate to someone I know will not action it, that could be seen as performative?

Yes, however I think this is down to the conversation between a line manager and a report when it comes to discussing the provided evidence. For example when filling the competencies out, a report offers the evidence "I helped PersonA and PersonB resolve this dispute". It's now down to the line manager to make sure that the dispute was _actually_ resolved, this could take the form of asking PersonA and PersonB for feedback on their report's handling of the dispute.

Here we're potentially sliding into trying to define what good line management and conflict resolution looks like, which isn't the job of the competencies.

I'm not sure if we've documented this anywhere (we maybe should), but for all of the competencies it's assumed that when a person is using something they did as evidence, they have done that thing _well_. For example if we take the competency "Regularly contributes openly to team meetings and encourages others to do so", we don't need to state that they've contributed in a useful way, that's assumed.